tests/conftest.py

```diff
@@ -442,6 +442,31 @@ def finalizer():
         Delete the project
         """
         for instance in instances:
+            try:
+                ocp_event = ocp.OCP(kind="Event", namespace=instance.namespace)
+                events = ocp_event.get()
+                event_count = len(events['items'])
+                warn_event_count = 0
+                for event in events['items']:
+                    if event['type'] == "Warning":
+                        warn_event_count += 1
+                log.info(
+                    (
+                        "There was %d events in %s ns before it's removal"
+                        " (our of which %d was of type Warning)."
```

Comment:
Change to:
" (out of which %d were of type Warning)."

Reply:
fixed